.github/workflows/docker-image-push.yml

```diff
@@ -0,0 +1,58 @@
+name: Build (and push) multi-arch Docker image
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    # We want a newer version of qemu: https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1815100
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile: [Dockerfile, alpine/Dockerfile]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Docker login
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+        run: echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin docker.io
+
+      - name: Setup buildx
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y qemu-user-static
+          docker buildx create --use
+
+      - name: Build Docker image
+        env:
+          PLATFORMS: linux/amd64,linux/arm64/v8,linux/arm/v7
+          DOCKER_REPOSITORY: thelounge/thelounge
+        run: |
+          TAG="$(git rev-parse --short HEAD)"
+          EXTRA_ARG=()
+          # Only push on regular push (!= pull_request)
+          if [ "${GITHUB_EVENT_NAME}" = "push" ] && grep -q "^refs/tags/" <<< "${GITHUB_REF}"; then
```

Comment:
Can't this if statement be removed because there's already an `if:` for the entire job?

Reply:
It almost seem like anything outside this `if` can be removed (apart from the `docker buildx build`)?

Reply:
We want to build for all commits but only push for tags.